libgalois/test/property-file-graph.cpp

```diff
@@ -96,9 +96,9 @@ TestTypesFromPropertiesCompareTypesFromStorage() {
   auto g2 = std::move(make_result.value());
 
   KATANA_LOG_VASSERT(
-      g2->GetNumNodeEntityTypes() != 0, "found {} entity types.",
+      g2->GetNumNodeEntityTypes() == 1, "found {} entity types.",
```

Comment:
A similar check is needed on line 75-77. Shouldn't the number of types be 2 given that kUnknownType is already there. Also, you should add a check to make sure that nodes don't have kUnknownType. In fact, you know the type ID of the new type as well. It should be 1 if you added 1 new type (besides kUnknownType).